scripts/qa_crowbarsetup.sh

```diff
@@ -5010,6 +5010,27 @@ function power_cycle_and_wait
         "node $m_hostname to power cycle"
 }
 
+function recover_if_fenced
+{
+    local machine=$1
+
+    # Check the machine has a problem reported by crowbar
+    if ! crowbar node_state status | grep $machine | grep -i "problem$"; then
+        return
+    fi
+
+    # Check the machine was fenced
+    if ! ssh $machine 'ls /var/spool/corosync/block_automatic_start'; then
+        return
+    fi
+
+    # Fencing recovery steps (fencing on reboot particular case)
+    # https://w3.suse.de/~aspiers/cloud/HA-training/HA-training-all-slides.pdf
+    ssh $machine 'rm /var/spool/corosync/block_automatic_start'
+    ssh $machine 'systemctl stop chef-client.service'
+    ssh $machine 'systemctl start crowbar_join.service'
```

Comment:
afaik `crowbar_join` is not started in the node after the reboot. We are trying to get the node to join the cluster again. This was directly extracted from the [HA guide](https://w3.suse.de/~aspiers/cloud/HA-training/HA-training-all-slides.pdf) (pages 108 and 109).